### PR TITLE
Feature/page titles

### LINKED ISSUE #602: Change title of page for different pages
Currently html title is static.

### PR DESCRIPTION
**Pull request by DavidMarchant**
Closes #602 when merged
Added a single method in `application_helper` to generate the title
The titles I went for are
```
Default ('Alces Flight Center') if not on a site specific page
If on a site but not a cluster: <site_name> - Alces Flight Center
If on a cluster homepage: <cluster_name> - Alces Flight Center
If on a cluster non-homepage: <name_of_the_tab> - <cluster_name> - Alces Flight Center
If viewing a component group: <name_of_the_component_group> - <cluster_name> - Alces Flight Center
```
How's this with people?
Apologies that this took me so long, it dealt with quite a broad section of the site much of which I was unfamiliar with 
Trello: https://trello.com/c/a8avcjvj/498-make-dynamic-page-titles

### CODE REVIEW

**Comment by DavidMarchant** [comment]
ah yeah, cheers! No that all makes sense. I felt this was very inflexible & inopportune as i was writing it, I should've asked really - thanks for the feedback. I'll re-do it now & move it to have a method in each class using `content_for`

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -67,6 +67,26 @@ def simple_format_if_needed(text)
     text.include?("\n") ? simple_format(text) : text
   end
 
+  def generate_title()
+    title = "Alces Flight Center"
+    if @scope.is_a? Cluster
```
[the remark this answers is not in the document]
Comment (DavidMarchant) [reply app/helpers/application_helper.rb]: Ah right yeah, that makes sense, I'll take it on board, thanks :man_teacher: 
Coincidently and somewhat annoyingly if it _had_ been Java I'd have probably done it right the first time - I'm still very much figuring out how these more weakly-typed _scripting-ish_ languages deal with a lot of the OOP stuff

**Comment by jamesremuscat** [comment]
> I'll re-do it now & move it to have a method in each class using content_for

Just for clarity: the `content_for` blocks belong in the view templates, not the models. :slightly_smiling_face:

**Comment by DavidMarchant** [comment]
Right! sorry, cheers :slightly_smiling_face:

**Pull request merged**